src/pages/settings/Security/TwoFactorAuth/CodesPage.js

```diff
@@ -73,7 +75,7 @@ function CodesPage(props) {
                     <View style={styles.mv3}>
                         <Text>{props.translate('twoFactorAuth.codesLoseAccess')}</Text>
                     </View>
-                    <View style={styles.twoFactorAuthCodesBox}>
+                    <View style={[styles.twoFactorAuthCodesBox, {paddingHorizontal: props.isSmallScreenWidth ? 40 : 60}]}>
```

Comment:
I think you should use it like this, to keep the project pattern:

```suggestion
                    <View style={[styles.twoFactorAuthCodesBox, props.isSmallScreenWidth ? styles.ph10 : styles.ph15]}>
```

And you'll need to create `styles.ph15`